src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -0,0 +1,222 @@
+from shutil import copytree
+from summarize import create_summary
+from merge_xmls import merge_xml
+from pathlib import Path
+from sys import version
+
+import xml.etree.ElementTree as ET
+
+import os
+
+logger = utils.get_logger('ConformanceRunner')
+
+OPENVINO_NAME = 'openvino'
+
+OMZ_REPO_URL = "https://github.com/openvinotoolkit/open_model_zoo.git"
+OMZ_REPO_BRANCH = "master"
+
+GTEST_PARALLEL_URL = "https://github.com/intel-innersource/frameworks.ai.openvino.ci.infrastructure.git"
+GTEST_PARALLEL_BRANCH = "master"
+
+API_CONFORMANCE_BIN_NAME = "apiConformanceTests"
+OP_CONFORMANCE_BIN_NAME = "conformanceTests"
+SUBGRAPH_DUMPER_BIN_NAME = "subgraphsDumper"
+
+NO_MODEL_CONSTANT = "NO_MODEL"
+
+def get_ov_path(ov_dir=None, is_bin=False):
+    if ov_dir is None or not os.path.exists(ov_dir):
+        try:
+            ov_dir = os.environ['INTEL_OPENVINO_DIR']
+        except:
+            ov_dir = os.path.abspath(os.getcwd())[:os.path.abspath(os.getcwd()).find(OPENVINO_NAME) + len(OPENVINO_NAME)]
+    if is_bin:
+        ov_dir = os.path.join(ov_dir, 'bin')
+        get_latest_dir = lambda path: sorted(Path(ov_dir).iterdir(), key=os.path.getmtime)[0]
+        ov_dir = os.path.join(get_latest_dir(ov_dir))
+        ov_dir = os.path.join(get_latest_dir(ov_dir))
+    return ov_dir
+
+
+
+def parse_arguments():
+    parser = ArgumentParser()
+
+    models_path_help = "Path to directory/ies contains models to dump subgraph (default way is download OMZ). If `--d=False` specify Conformance IRs directory"
+    device_help = "Specify target device. Default value is CPU"
+    ov_help = "OV binary files path"
+    working_dir_help = "Specify working directory"
+    type_help = "Specify conformance type: OP or API"
+    dump_conformance_help = "Set 'True' if you want to create Conformance IRs from custom models. Default value is 'False'"
+
+    parser.add_argument("-m", "--models_path", help=models_path_help, type=str, required=False, default=NO_MODEL_CONSTANT)
+    parser.add_argument("-d", "--device", help= device_help, type=str, required=False, default="CPU")
+    parser.add_argument("-ov", "--ov_path", help=ov_help, type=str, required=False, default=get_ov_path())
+    parser.add_argument("-w", "--working_dir", help=working_dir_help, type=str, required=False, default='temp')
+    parser.add_argument("-t", "--type", help=type_help, type=str, required=False, default="OP")
+    parser.add_argument("-s", "--dump_conformance", help=dump_conformance_help, type=int, required=False, default=1)
+
+    return parser.parse_args()
+
+class Conformance:
+    def __init__(self, device:str, model_path:str, ov_path:str, type:str, working_dir:str):
+        self._device = device
+        self._model_path = model_path
+        self._ov_path = ov_path
+        self._ov_bin_path = get_ov_path(self._ov_path, True)
+        self._working_dir = working_dir
+        if not (type == "OP" or type == "API"):
+            raise Exception("Incorrect conformance type")
```

Comment:
Maybe I'm too C++sed, but exceptions in a "constructor" scares me. Just want to highlight, need to check python's understanding of this kind of errors. Especially because we don't catch it in code below.